earth_enterprise/rpms/build.gradle

```diff
@@ -43,6 +47,21 @@ apply plugin: "nebula.ospackage"
 
 import org.opengee.shell.GeeCommandLine
 
+// Varables used for specifying location of two files
+// 'gee_long_version.txt' and 'gepgcommon'
+def working_dir = "../src/"
+
+// Check for either "-PRELEASE" or "-POPTIMIZE" command line parameters.
+// Return the correct directory based on which paramater is used.
+def native_x86 = {
```

Comment:
Minor: Could you include something the name of this function that indicates that it is the name of directory, and it's build directory?